docs/quick-start.md

```diff
@@ -19,11 +19,20 @@ You'll get to know:
 - Get familiar with core concepts as [Application](./getting-started/core-concept#application), [Environment](getting-started/core-concept#environment) and [Target](getting-started/core-concept#target)
 - Finished an application delivery by operating VelaUX
 
+## Get password and login
+
+After install VelaUX, you need to log in. Default username is "admin" and you can get the password using command below
+```shell
+vela logs -n vela-system --name apiserver addon-velaux | grep "init admin user"
```

Comment:
It's can not work in beta.2 version.

Reply:
Let's hold this until we publish a new version?

Reply:
changed to `initialized admin username` https://github.com/oam-dev/kubevela/pull/3563/files